packages/components/src/PassengerCard/PassengerCard.js

```diff
@@ -0,0 +1,135 @@
+// @flow strict
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { Text, StyleSheet, Icon } from '@kiwicom/universal-components';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+
+import BagInformation from './BagInformation';
+import Separator from '../separator/Separator';
+
+type Bag = {|
+  +count: number,
+  +type: string,
+|};
+
+type Props = {|
+  +blank: boolean,
+  +name: string,
+  +gender: 'female' | 'male' | 'other',
+  +nationality: string,
+  +dateOfBirth: string,
+  +id: string,
+  +insurance: string,
+  +bags: Array<Bag>,
+  +passengerCount: number,
+|};
+
+export default function PassengerCard(props: Props) {
+  const newPassengerTitle = `${props.passengerCount}. Passenger`;
+  const preffix =
```

Comment:
Prefix should be with one `f` 😉

Reply:
Oops😅 well its gonna be renamed to title now